azure-pipelines/continous-delivery/common.yml

```diff
@@ -0,0 +1,76 @@
+# File: azure-pipelines\continous-delivery\common.yml
+# Description: Assemble & publish common release to internal feed
+
+name: 0.$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+- dev
```

Comment:
So this is doing the trick here? I assume having this here means the pipeline will run on every push to dev?

Reply:
Correct